intersections/build.boot

```diff
@@ -0,0 +1,113 @@
+(set-env!
+ :repositories #(conj % '["bintray" {:url "http://dl.bintray.com/nitram509/jbrotli"}])
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.6.0" :scope "test"]
+                 [nha/boot-uglify "0.0.7" :scope "test"]])
+
+(def +version+ "1.0.0-0")
+
+(task-options!
+ pom  {:project     'cljsjs/intersections
```

Comment:
Package name should be same as in other package manages (npm, bower) or github: "kld-intersections".